src/pages/home.tsx

```diff
@@ -0,0 +1,37 @@
+import Head from 'next/head';
+
+export default function HomePage() {
+  return (
+    <section>
+      <Head>
+        <title>Wars World Home Page</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <div className="@rounded-lg @bg-black/70 @center @p-8 @max-w-3xl isProgress">
```

Comment:
Super nitpicky. should be `inProgress`, not `isProgress`
```suggestion
      <div className="@rounded-lg @bg-black/70 @center @p-8 @max-w-3xl inProgress">
```

Reply:
Got it, getting this fixed. It appears I've missed the className in the spreadsheet as well (Line 33 of layout.scss). I'll change it there as well.